### ISSUE TITLE
Desktop App: Focus visible styles not showing in almost all settings menus

### ISSUE THREAD

**Issue by jcampuza** [anomalyco/opencode#12611]
### Description

In the desktop settings menu almost no components have styles for when focus-visible should be showing - i.e keyboard navigation.

It seems this was potentially unintentional after a refactor of some focus styles. Some styles seem to have been added to specifically remove some focus styles when using the mouse, but it results in almost nothing being shown and no indication as to the active element when using keyboard.

### OpenCode version

Desktop: 1.1.53

### Steps to reproduce

Open the settings menu with the keyboard (cmd+,)
Use the keyboard to navigate around with tab+navigate
No styles will be shown on almost anything including the following:
- The tab navigation menu on the side
- Switch components
- Select components

### Screenshot and/or share link

In the following gif you'll see I have keycastr running and am using the keyboard, but nothing shows a focus/active style.

![Image](https://github.com/user-attachments/assets/2c3082b2-6152-414b-9637-0df52b8e9899)

### Operating System

macOS
